ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -1749,6 +377,94 @@
 			shellPath = /bin/sh;
 			shellScript = "if [[ $CONFIGURATION == \"Release\" && ! `git describe --tags` =~ '-g' ]]; then\n  export SENTRY_PROPERTIES=sentry.properties\n  ../node_modules/@sentry/cli/bin/sentry-cli upload-dsym\nelse\n  echo [Upload Debug Symbols to Sentry] Skipping sentry\nfi\n";
 		};
+		8BC51697242D4EF80019892C /* Start Metro */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Start Metro";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "export RCT_METRO_PORT=\"${RCT_METRO_PORT:=8081}\"\necho \"export RCT_METRO_PORT=${RCT_METRO_PORT}\" > \"${SRCROOT}/../node_modules/react-native/scripts/.packager.env\"\nif [ -z \"${RCT_NO_LAUNCH_PACKAGER+xxx}\" ] ; then\n  if nc -w 5 -z localhost ${RCT_METRO_PORT} ; then\n    if ! curl -s \"http://localhost:${RCT_METRO_PORT}/status\" | grep -q \"packager-status:running\" ; then\n      echo \"Port ${RCT_METRO_PORT} already in use, packager is either not running or not running correctly\"\n      exit 2\n    fi\n  else\n    open \"$SRCROOT/../node_modules/react-native/scripts/launchPackager.command\" || echo \"Can't start packager automatically\"\n  fi\nfi\n";
```

Comment:
So, I think the reason this is newly necessary is that `React.xcodeproj` has its own "Start Packager" build phase, within the "React" target.

I took a look at that one for comparison. It has some helpful features this version lacks, which would be nice to incorporate. (Notably, it avoids bothering to launch the packager for a release build.) ... ha, apparently I was looking at a v0.60 version of React.xcodeproj, oops :slightly_smiling_face: Anyway, that is a good feature.

This is an unfriendly spot to have to edit it, though. It'd be much cleaner as a separate script file.

Anyway, doing anything about that can be a followup. (And then a good PR upstream, I think!)

Reply:
Noted, good idea!